### ISSUE TITLE
[BUG] Dash documentation on Slider marks

### ISSUE THREAD

**Issue by grofte** [plotly/dash-docs#613]
Great software.

**Describe your context**
On the Dash website there is an explanation of the Slider component https://dash.plot.ly/dash-core-components/slider
Under the section "Marks and Steps" it says "[...] the slider will only be able to select values that have been predefined by the marks." 
I'm on Chrome under Windows 10 but I doubt that's relevant. 

**Describe the bug**

When moving the slider you can select values not defined in `marks`. Additionally, you can't select 7.65, a value defined in `marks`. 

**Expected behavior**

I expected the slider to behave as described, i.e. limit the selectable values and have all defined values be available. 

**Screenshots**

![dash_plotly_slider](https://user-images.githubusercontent.com/7976840/62218200-45e6a100-b3ac-11e9-956d-521d61014a9d.png)

**Comment by alexcjohnson** [comment]
Thanks @grofte 
The issue is that `step` has a default of 1 (though this is never explicitly stated anywhere in our own docs - you have to go look at the component we wrapped, https://github.com/schrodinger/rc-slider to see that) and you have to explicitly say `step=None` to get the described behavior, which puts the text in contradiction with the example code.

**Comment by grofte** [comment]
Cool @alexcjohnson  
That should be an easy fix then. 

I wasn't sure if this was the right place to report documentation problems though?

**Comment by alexcjohnson** [comment]
@grofte don't worry too much about where to report - I'd much rather get a report in an adjacent repo than not get it at all. And github has now made it easy to move an issue between repos, so I just moved it where I think the work will be done. Thanks again for the report!

**Comment by grofte** [comment]
Oh, I didn't even realise we were in a different repo now. Fancy!

**Comment by grofte** [comment]
Actually, I guess I could just clone and fix it myself?  
And there's some stuff in the Dash DataTables tutorial that I think could use some TLC as well.

**Comment by alexcjohnson** [comment]
Absolutely, we would love a PR to this repo to address any deficiencies you see!

**Comment by charlesdwright** [comment]
Has anyone noticed (it's all over the examples) that the last mark (10 °F) wraps the characters after the space to a new line?

This is no bueno.  One of those 80/20 Rule things that makes you pull your hair out when you're burning the midnight oil to finish your project.

Otherwise, love the library & thx.
